subprojects/plugins/src/integTest/groovy/org/gradle/java/compile/AbstractJavaGroovyCompileAvoidanceIntegrationSpec.groovy

```diff
@@ -210,21 +210,6 @@ include 'a', 'b'
         succeeds ":b:${language.compileTaskName}"
         executedAndNotSkipped ":a:${language.compileTaskName}", ":b:${language.compileTaskName}"
 
-        when:
-        // add public constructor to replace the default, should not change
```

Comment:
* implicit and explicit default constructors don't generate the same bytecode anymore (implicit one is marked as `@Generated` now), see: https://issues.apache.org/jira/browse/GROOVY-9051
* static initializers generate and empty if statement. This seems like a bug, opened:
https://issues.apache.org/jira/browse/GROOVY-9166

Reply:
TODO separate Groovy tests